db/institutions.yml

```diff
@@ -369,20 +371,13 @@ uiuc:
   name: University of Illinois at Urbana-Champaign
   state: IL
   zip: 61820-5711
-northwesternu:
-  country: USA
-  aka:
-  - Northwestern
-  - northwestern
-  city: Evanston
-  name: Northwestern University
-  state: IL
-  zip: '60208'
 ghentu:
-  aka: [University of Ghent, Universiteit Gent, UGent, State University of Ghent, Rijksuniversiteit Gent, RUG]
+  aka: [University of Ghent, Universiteit Gent, UGent, State University of Ghent,
+    Rijksuniversiteit Gent, RUG]
   city: Ghent
```

Comment:
I've always wanted to learn the indentation rule in json. Could you please explain when we need 2 spaces (like here) and when we need 6 spaces as the aka filed inside department dict (line 312 in this PR)? Thanks so much.